client/app/routes/post.tsx

```
@@ -0,0 +1,36 @@
+      postData: postData.data.post,
+      commentData: commentData.data
+    };
+  }
+}
+
+
+export default function PostRoute({loaderData}: Route.ComponentProps){
+return <>
+    <Post post={(loaderData?.postData) as PostData}/>
+    {loaderData?.commentData?loaderData?.commentData.map((comment: CommentData)=>{
+        return <Comment comment={comment}/>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)